core/engine/src/module/mod.rs

```diff
@@ -625,16 +624,17 @@ impl Module {
 
     /// Get an exported function, typed, from the module.
     #[inline]
+    #[allow(clippy::needless_pass_by_value)]
     pub fn get_typed_fn<A, R>(
         &self,
-        name: JsStr<'_>,
+        name: JsString,
```

Comment:
Nit: This should probably be a compatible `impl Into<PropertyKey>` here, similar to `Module::get_value`.

Reply:
`impl Into<PropertyKey>` includes integer and `JsSymbol`, which are illegal here in `ecmascript`, I think it will mislead user.